wallets/electrum/src/main/java/bisq/wallets/electrum/notifications/ElectrumNotifyApi.java

```diff
@@ -36,22 +37,22 @@ public interface Listener {
         void onAddressStatusChanged(String address, String status);
     }
 
-    private static final CopyOnWriteArrayList<Listener> sListeners = new CopyOnWriteArrayList<>();
+    private static final List<Listener> listeners = new CopyOnWriteArrayList<>();
 
     @POST
     @Consumes("application/json")
     public Response notifyEndpoint(@Parameter(required = true) ElectrumNotifyRequest request) {
         log.info(request.toString());
-        sListeners.forEach(listener -> listener.onAddressStatusChanged(request.getAddress(), request.getStatus()));
+        listeners.forEach(listener -> listener.onAddressStatusChanged(request.getAddress(), request.getStatus()));
         return Response.ok().entity("SUCCESS").build();
```

Comment:
return "SUCCESS"  would do the same. You notation is usefull if you may want to send back anything other than Response.ok() in special case.